src/main/main.ts

```diff
@@ -58,21 +58,20 @@ export { DeviceCredential, LoginWithWebAuthnParams, SignupWithWebAuthnParams } f
 export interface Config {
   clientId: string
   domain: string
+  /**
+   * Used to define which Google provider variant to use with Google One Tap
+   * @default "default"
+   * */
+  googleVariant?: string
```

Comment:
Generify to `variant` ?

Reply:
Je me suis posé la question, mais deux choses m'ont fait opter pour cette solution : 
- c'est en association avec le propriété `googleClientId`
- il n'y a que le clientId de Google qui est transmis ainsi (ce qui devrait probablement être fait autrement d'ailleurs mais c'est encore autre chose...)